juriscraper/opinions/united_states/state/delaware.py

```diff
@@ -18,46 +19,60 @@ def __init__(self, *args, **kwargs):
         # Note that we can't do the usual thing here because 'del' is a Python keyword.
         self.court_id = "juriscraper.opinions.united_states.state.del"
         self.should_have_results = True
+        self.status = "Published"
+
+    def _process_html(self):
+        """Process the html and extract out the opinions
+
+        :return: None
+        """
+        for row in self.html.xpath("//table/tr"):
+            self.cases.append(
+                {
+                    "name": row.xpath("./td[1]/a/text()")[0].strip(),
+                    "date": row.xpath("./td[2]/text()")[0].strip(),
+                    "docket": row.xpath("./td[3]/a/text()")[0].strip(),
+                    "url": row.xpath("./td[1]/a/@href")[0].strip(),
+                    "judge": " ".join(row.xpath("./td[6]/text()")),
+                }
+            )
+
+    def extract_from_text(self, scraped_text: str) -> dict:
+        """Extract lower court from the scraped text.
+
+        :param scraped_text: The text to extract from.
+        :return: A dictionary with the metadata.
+        """
+
+        cleaned_text = "\n".join(
+            line.rsplit("§", 1)[-1] if "§" in line else line
+            for line in scraped_text[:1500].splitlines()
+        )
+
+        pattern = re.compile(
+            r"Court\s+Below[–—\-:]\s*(?P<lower_court>.*?)\n\s*\n",
```

Comment:
I only found that in one case it returned a "the" before the court name:

```
Adding new item:
case_dates: "2025-09-15"
case_names: "Allergan Sales, LLC v. Sofregen Medical, Inc."
download_urls: "http://courts.delaware.gov/Opinions/Download.aspx?id=384860"
precedential_statuses: "Published"
blocked_statuses: "False"
date_filed_is_approximate: "False"
docket_numbers: "28, 2025"
judges: "Griffiths J."
case_name_shorts: ""

Values obtained by Site.extract_from_text:
Docket
    appeal_from_str: "the Superior Court of the State of Delaware"
OriginatingCourtInformation
    docket_number: "N20C-03-319"
```
You only have to take this case into account for regex: `Court Below–the`